### DISCUSSION TITLE
Tabs multiline?

### DISCUSSION THREAD

**Discussion by faizananwerali** [filamentphp/filament#8546] [Help]
I have a very bad layout on my Mac M1 Pro.

Is there a way that tabs can be multiline?

<img width="1117" alt="Screenshot 2023-09-16 at 8 58 37 PM" src="https://github.com/filamentphp/filament/assets/12691366/c2b98a66-3d7e-4ea1-bf8b-d0fcce483d1f">

Currently, I've fixed it by modifying CSS in `resources/css/filament/admin/theme.scss`.

```scss
.fi-tabs {
    @apply flex-wrap justify-center;
}
```
<img width="1122" alt="Screenshot 2023-09-16 at 9 00 57 PM" src="https://github.com/filamentphp/filament/assets/12691366/d93b54c2-8b3d-4ae6-b12b-27d9242c4169">

But this isn't the proper solution. Does anyone know the proper solution?

**Comment by leandrocfe** [comment]
https://github.com/filamentphp/filament/discussions/8547 related question?

**Reply by faizananwerali** [reply]
Not related. Using CSS is my workaround, not an actual and proper solution.
